troyleach/divegalaxsea-ui
# Twl footer

### How to test drive this PR

- checkout branch
- `npm i`
- spin up a server
- Bring up the app should see something similar too

![image](https://user-images.githubusercontent.com/7662875/69207389-3a62e580-0b15-11ea-800d-b17ab40052ff.png)
